.eslintrc.json

```diff
@@ -1,7 +1,22 @@
 {
     "extends": "eslint",
+    "env": {
+        "es2020": true
+    },
+    "parserOptions": {
+        "sourceType": "module",
+        "ecmaVersion": "2020"
+    },
     "env": {
         "es6": true,
         "node": true
```

Comment:
This is now a duplicate `"env"` key. I think we can just remove this one, the other `"env"` enables `es2020`, and `eslint-plugin-node` enables appropriate globals for esm and cjs files.

Reply:
Sorry, missed that. Done.